### ISSUE TITLE
[1.3-rc.8] Meteor is unable to downgrade to 1.2.1

### ISSUE THREAD

**Issue by awwx** [meteor/meteor#6577]
Wasn't sure if this is expected or not:

```
$ meteor create x --release 1.3-rc.8
Created a new Meteor app in 'x'.

To run your new app:
  cd x
  meteor

If you are new to Meteor, try some of the learning resources here:
  https://www.meteor.com/learn

$ cd x
$ meteor update --release 1.2.1
Update to release METEOR@1.2.1 is impossible:
While selecting package versions:
error: Conflict: Constraint meteor@1.1.12-rc.8 is not satisfied by meteor 1.1.10.
Constraints on package "meteor":
* meteor@=1.1.10 <- top level
* meteor@1.1.7 <- meteor-base 1.0.1
* meteor@1.1.7 <- underscore 1.0.4 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- underscore 1.0.4 <- binary-heap 1.0.4 <- mongo 1.1.3
* meteor@1.1.7 <- underscore 1.0.4 <- meteor-base 1.0.1
* meteor@1.1.10 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.10 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- logging 1.0.8 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- logging 1.0.8 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- ejson 1.0.7 <- check 1.1.0 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- ejson 1.0.7 <- check 1.1.0 <- babel-compiler 5.8.24_1 <- ecmascript 0.1.6
* meteor@1.1.7 <- ejson 1.0.7 <- check 1.1.0 <- mongo 1.1.3
* meteor@1.1.7 <- ejson 1.0.7 <- mongo 1.1.3
* meteor@1.1.7 <- base64 1.0.4 <- ejson 1.0.7 <- check 1.1.0 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- base64 1.0.4 <- ejson 1.0.7 <- check 1.1.0 <- babel-compiler 5.8.24_1 <- ecmascript 0.1.6
* meteor@1.1.7 <- base64 1.0.4 <- ejson 1.0.7 <- check 1.1.0 <- mongo 1.1.3
* meteor@1.1.7 <- base64 1.0.4 <- ejson 1.0.7 <- mongo 1.1.3
* meteor@1.1.7 <- routepolicy 1.0.6 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- routepolicy 1.0.6 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- routepolicy 1.0.6 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- routepolicy 1.0.6 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- htmljs 1.0.5 <- blaze 2.1.3 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- deps 1.0.9 <- htmljs 1.0.5 <- blaze 2.1.3 <- blaze-html-templates 1.0.1
* meteor@1.1.9 <- tracker 1.0.9
* meteor@1.1.7 <- html-tools 1.0.5 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <-
meteor-base 1.0.1
* meteor@1.1.7 <- html-tools 1.0.5 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- blaze-tools 1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0
<- meteor-base 1.0.1
* meteor@1.1.7 <- blaze-tools 1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- spacebars 1.0.7 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- blaze 2.1.3 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- jquery 1.11.4
* meteor@1.1.10 <- check 1.1.0 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.10 <- check 1.1.0 <- babel-compiler 5.8.24_1 <- ecmascript 0.1.6
* meteor@1.1.10 <- check 1.1.0 <- mongo 1.1.3
* meteor@1.1.7 <- observe-sequence 1.0.7 <- blaze 2.1.3 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- mongo-id 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- mongo-id 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- mongo-id 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- mongo-id 1.0.1 <- minimongo 1.0.10 <- mongo 1.1.3
* meteor@1.1.7 <- mongo-id 1.0.1 <- mongo 1.1.3
* meteor@1.1.7 <- id-map 1.0.4 <- binary-heap 1.0.4 <- mongo 1.1.3
* meteor@1.1.10 <- random 1.0.5 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.10 <- random 1.0.5 <- minimongo 1.0.10 <- mongo 1.1.3
* meteor@1.1.10 <- random 1.0.5 <- mongo 1.1.3
* meteor@1.1.10 <- ecmascript 0.1.6
* meteor@1.1.7 <- babel-compiler 5.8.24_1 <- ecmascript 0.1.6
* meteor@1.1.7 <- babel-runtime 0.1.4 <- ecmascript 0.1.6
* meteor@1.1.10 <- ecmascript-runtime 0.2.6 <- ecmascript 0.1.6
* meteor@1.1.10 <- promise 0.5.1 <- ecmascript 0.1.6
* meteor@1.1.7 <- diff-sequence 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- diff-sequence 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- diff-sequence 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- diff-sequence 1.0.1 <- minimongo 1.0.10 <- mongo 1.1.3
* meteor@1.1.7 <- diff-sequence 1.0.1 <- mongo 1.1.3
* meteor@1.1.7 <- reactive-var 1.0.6
* meteor@1.1.7 <- ui 1.0.8 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- webapp-hashing 1.0.5 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- webapp-hashing 1.0.5 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- retry 1.0.4 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.10 <- ddp-common 1.2.2 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.10 <- ddp-common 1.2.2 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.10 <- ddp-common 1.2.2 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- callback-hook 1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- callback-hook 1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- callback-hook 1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- callback-hook 1.0.4 <- mongo 1.1.3
* meteor@1.1.9 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.9 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.9 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.9 <- minimongo 1.0.10 <- mongo 1.1.3
* meteor@1.1.7 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- mongo 1.1.3
* meteor@1.1.7 <- geojson-utils 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- geojson-utils 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- geojson-utils 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- geojson-utils 1.0.4 <- minimongo 1.0.10 <- mongo 1.1.3
* meteor@1.1.7 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.10 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.10 <- mongo 1.1.3
* meteor@1.1.7 <- npm-mongo 1.4.39_1 <- mongo 1.1.3
* meteor@1.1.7 <- binary-heap 1.0.4 <- mongo 1.1.3
* meteor@1.1.7 <- http 1.1.1 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- url 1.0.5 <- http 1.1.1 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- reload 1.1.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- reload 1.1.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* meteor@1.1.7 <- reload 1.1.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* meteor@1.1.7 <- reload 1.1.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* meteor@1.1.7 <- mobile-experience 1.0.1
* meteor@1.1.7 <- launch-screen 1.0.4 <- mobile-experience 1.0.1
* meteor@1.1.7 <- fastclick 1.0.7 <- mobile-experience 1.0.1
* meteor@1.1.7 <- mobile-status-bar 1.0.6 <- mobile-experience 1.0.1
* meteor@1.1.7 <- blaze-html-templates 1.0.1
* meteor@1.1.10 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* meteor@1.1.9 <- caching-html-compiler 1.0.2 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- caching-compiler 1.0.0 <- caching-html-compiler 1.0.2 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- templating-tools 1.0.0 <- caching-html-compiler 1.0.2 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- templating-tools 1.0.0 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* meteor@1.1.7 <- minifiers 1.1.7 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <-
meteor-base 1.0.1
* meteor@1.1.7 <- minifiers 1.1.7 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* meteor@1.1.7 <- minifiers 1.1.7 <- templating-tools 1.0.0 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* meteor@1.1.12-rc.8 <- standard-minifier-css 1.0.4-rc.8
* meteor@1.1.12-rc.8 <- minifier-css 1.1.9-rc.8 <- standard-minifier-css 1.0.4-rc.8
* meteor@1.1.12-rc.8 <- standard-minifier-js 1.0.4-rc.8
* meteor@1.1.12-rc.8 <- minifier-js 1.1.9-rc.8 <- standard-minifier-js 1.0.4-rc.8
* meteor@1.1.10 <- es5-shim 4.1.14
* meteor@1.1.7 <- autopublish 1.0.4
* meteor@1.1.7 <- insecure 1.0.4

Conflict: Constraint underscore@1.0.6-rc.8 is not satisfied by underscore 1.0.4.
Constraints on package "underscore":
* underscore@=1.0.4 <- top level
* underscore@1.0.4 <- meteor-base 1.0.1
* underscore@1.0.4 <- meteor 1.1.10 <- autopublish 1.0.4
* underscore@1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* underscore@1.0.4 <- logging 1.0.8 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- logging 1.0.8 <- webapp 1.2.3 <- meteor-base 1.0.1
* underscore@1.0.4 <- ejson 1.0.7 <- check 1.1.0 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- ejson 1.0.7 <- check 1.1.0 <- babel-compiler 5.8.24_1 <- ecmascript 0.1.6
* underscore@1.0.4 <- ejson 1.0.7 <- check 1.1.0 <- mongo 1.1.3
* underscore@1.0.4 <- ejson 1.0.7 <- mongo 1.1.3
* underscore@1.0.4 <- routepolicy 1.0.6 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- routepolicy 1.0.6 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- routepolicy 1.0.6 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- routepolicy 1.0.6 <- webapp 1.2.3 <- meteor-base 1.0.1
* underscore@1.0.4 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* underscore@1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base
1.0.1
* underscore@1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* underscore@1.0.4 <- blaze-tools 1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push
1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- blaze-tools 1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* underscore@1.0.4 <- spacebars 1.0.7 <- blaze-html-templates 1.0.1
* underscore@1.0.4 <- blaze 2.1.3 <- blaze-html-templates 1.0.1
* underscore@1.0.4 <- check 1.1.0 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- check 1.1.0 <- babel-compiler 5.8.24_1 <- ecmascript 0.1.6
* underscore@1.0.4 <- check 1.1.0 <- mongo 1.1.3
* underscore@1.0.4 <- observe-sequence 1.0.7 <- blaze 2.1.3 <- blaze-html-templates 1.0.1
* underscore@1.0.4 <- id-map 1.0.4 <- binary-heap 1.0.4 <- mongo 1.1.3
* underscore@1.0.4 <- random 1.0.5 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- random 1.0.5 <- minimongo 1.0.10 <- mongo 1.1.3
* underscore@1.0.4 <- random 1.0.5 <- mongo 1.1.3
* underscore@1.0.4 <- diff-sequence 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- diff-sequence 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- diff-sequence 1.0.1 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- diff-sequence 1.0.1 <- minimongo 1.0.10 <- mongo 1.1.3
* underscore@1.0.4 <- diff-sequence 1.0.1 <- mongo 1.1.3
* underscore@1.0.4 <- webapp-hashing 1.0.5 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- webapp-hashing 1.0.5 <- webapp 1.2.3 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- retry 1.0.4 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-common 1.2.2 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-common 1.2.2 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-common 1.2.2 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- callback-hook 1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- callback-hook 1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- callback-hook 1.0.4 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- callback-hook 1.0.4 <- mongo 1.1.3
* underscore@1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- minimongo 1.0.10 <- mongo 1.1.3
* underscore@1.0.4 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base
1.0.1
* underscore@1.0.4 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- ddp-server 1.2.2 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- mongo 1.1.3
* underscore@1.0.4 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- mongo 1.1.3
* underscore@1.0.4 <- binary-heap 1.0.4 <- mongo 1.1.3
* underscore@1.0.4 <- http 1.1.1 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- url 1.0.5 <- http 1.1.1 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- reload 1.1.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- reload 1.1.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- livedata 1.0.15 <- meteor-base 1.0.1
* underscore@1.0.4 <- reload 1.1.4 <- ddp-client 1.2.1 <- ddp 1.2.2 <- meteor-base 1.0.1
* underscore@1.0.4 <- reload 1.1.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* underscore@1.0.4 <- caching-html-compiler 1.0.2 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* underscore@1.0.4 <- templating-tools 1.0.0 <- caching-html-compiler 1.0.2 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* underscore@1.0.4 <- templating-tools 1.0.0 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* underscore@1.0.4 <- minifiers 1.1.7 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- autoupdate 1.2.4 <- hot-code-push 1.0.0
<- meteor-base 1.0.1
* underscore@1.0.4 <- minifiers 1.1.7 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.3 <- meteor-base 1.0.1
* underscore@1.0.4 <- minifiers 1.1.7 <- templating-tools 1.0.0 <- templating 1.1.5 <- blaze-html-templates 1.0.1
* underscore@1.0.6-rc.8 <- minifier-css 1.1.9-rc.8 <- standard-minifier-css 1.0.4-rc.8
* underscore@1.0.6-rc.8 <- minifier-js 1.1.9-rc.8 <- standard-minifier-js 1.0.4-rc.8

This project is at the latest release which is compatible with your current package constraints.
```

**Comment by zol** [comment]
Dupe of #6366 . Will be fixed by #6431

**Comment by awwx** [comment]
Thanks @zol !
